prow/config.yaml

```diff
@@ -201,6 +201,7 @@ tide:
     - do-not-merge/work-in-progress
     reviewApprovedRequired: true
   - repos:
+    - kubernetes/kube-openapi
```

Comment:
You don't _have_ to do this if you don't want to, but I would suggest maybe moving into the larger tide query above? 

I can't tell why this separate query exists, is listening to fewer `do-not-merge/*` labels really a needed thing?

lgtm otherwise

Reply:
If it's OK with you, I think we'll start off with the smaller tide query. If the need arises, we can move to the larger tide query in the future.